src/phases/query/IDHack.yml

```diff
@@ -12,7 +12,7 @@ IDHackObjectIdTemplate:
       Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: {unused: "please specify in which phases this actor should be active."}}}]
       NopInPhasesUpTo: {^Parameter: {Name: "NopInPhasesUpTo", Default: 99}}
       PhaseConfig:
-        Duration: 1 minute
```

Comment:
Can you capture in a comment here that our testing suggested that there was a lot more noise for approximately 20 seconds before the performance stabilized? I guess duplicate it everywhere that you changed here?

Reply:
done :)